src/app/components/fireside/stream/video/video.ts

```diff
@@ -54,4 +77,28 @@ export default class AppFiresideStreamVideo extends Vue {
 			}
 		}, 0);
 	}
+
+	@Watch('pausedFrameData')
+	private async onFrameDataChange() {
+		if (this.pausedFrameData) {
+			const {
+				$refs: { canvas },
+				pausedFrameData: { width, height },
+			} = this;
+
+			canvas.width = width;
+			canvas.height = height;
+			const context = canvas.getContext('2d')!;
+			context.putImageData(this.pausedFrameData, 0, 0, 0, 0, width, height);
```

Comment:
There is no way this could have transparency right? So we wouldn't need to clear what's already on the canvas?

Reply:
It shouldn't, but I changed it to clear the old data just in case.